exercises/sum.cpp

```diff
@@ -6,3 +6,30 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+using namespace std;
+
+
+int sum(int a,int b){
+
+	return (a+b);
+
+}
+
+int main(){
+
+	int a,b;
+
+	cout << "Insert the first number" << endl;	
+	cin >> a;
+	cout << "Insert the second number" << endl;
+	cin >> b;
+
+	cout << "Sum of " << a << " and " << b << endl;
+	int resSum=sum(a,b);
+
+	return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```